service/quote-server/README.md

```diff
@@ -0,0 +1,88 @@
+# Service: CCNP Quote Service
+
+This service will provide quote generated by underlying TEE platform for remote attestation service to verify the integrity and confidentiality of the trusted computing environment and required software environment.
+
+## Introduction
+
+This service provides functionality to fetch quote of underlying TEE platform with nonce as mandatory input and a base64 encoded user data as optional input. The nonce and optional user data will be digested and added into quote for remote attestation to verify the freshness of the quote and the user specified data.
+
+The quote server use Unix domain socket based gRPC to serve the client SDK and the proto buffer message is as bellow:
+
+```
+message HealthCheckRequest {
+  string service = 1;
+}
+
+message HealthCheckResponse {
+  enum ServingStatus {
+    UNKNOWN = 0;
+    SERVING = 1;
+    NOT_SERVING = 2;
+    SERVICE_UNKNOWN = 3;
+  }
+  ServingStatus status = 1;
+}
+
+service GetQuote {
+    rpc GetQuote (GetQuoteRequest) returns (GetQuoteResponse);
+}
+
+message GetQuoteRequest {
+   string user_data = 1;
+   string nonce = 2;
+}
+
+message GetQuoteResponse {
+    string quote = 1;
+    string quote_type = 2;
+}
+
+```
+
+## Installation
+The quote service can be deployed as either DaemonSet or sidecar in different user scenarios.
+
+### Build docker image
+The Dockerfile for the service can be found under `container/quote-server` directory. Use the following command to build the image:
+
+```
+docker build -t ccnp_get_quote:0.1 -f container/quote-server/Dockerfile .
+```
+
+> Note: if you are using containerd as the default runtime for kubernetes. Please remember to use the following command to import the image into containerd:
```

Comment:
if you are using containerd as the default runtime for kubernetes, don't forget to use the following commands to import the image into containerd first: